test/monitor.js

```diff
@@ -10,17 +10,17 @@ describe('monitor', function () {
         });
     });
 
-    describe('when created', function () {
-        it('should the plugins be empty', function () {
```

Comment:
The scenario should not be mirroring the implementation *(monitor.plugins.should.be.empty)